blog/2018-05-27-openhab-2-3-release.md

```diff
@@ -0,0 +1,53 @@
+
+## Localisation
+
+A lot of effort went into translation work during the past months: Not only the openHAB dashboard, but also HABPanel and the Android app have been translated into many languages by community members all over the world. We are using the [Crowdin service](https://crowdin.com/projects?q=openhab) for this, which makes it really easy for people to contribute - it is a perfect start for actively participating in the community!
+
+<!--{:.center}-->
+<img class="img-responsive" src="/uploads/translations.png" alt="">
+<small>Languages covered by HABPanel</small>
+
+## Units of Measurement
+
+Another great feature that makes it much easier to use openHAB all over the world is the new support for units on numbers. This make it now possible to e.g. use imperial units even if your sensors deliver values in the metric system. You can thus have all your temperatures shown as `°F`, completely independently from the hardware you are using - the conversion is automatically done by the framework. This not only works for simple conversions like `°C` to `°F`, but also for `hPa` to `mmHg` or similar things. Likewise, you can even decide to have different representations within your own measurement system. So while your sensor might deliver data in `m/s`, you can simply define that this is shown as `km/h` in your user interface.
+
+<!--{:.center}-->
+<img class="img-responsive" src="/uploads/uom.png" alt="">
+
+The real power of the unit support can be seen when being used within rules: Calculations can now be done unit-safe, thus [unit desasters](http://mentalfloss.com/article/25845/quick-6-six-unit-conversion-disasters) can be avoided. Even if you add `23|°F + 82|°F + 302|K` or `5|m + 30|cm + 0.01|mi` within a rule (note that the `|` is used as a delimiter between the value and its unit in rules), the result is always correctly calculated. You can find more technical background information about this feature in the [Eclipse SmartHome blog](https://www.eclipse.org/smarthome/blog/2018/02/22/units-of-measurement.html).
+
+## New Integrations
+
+We have a new record for the number of new add-ons for a release: openHAB 2.3 brings 25 new add-ons to the existing pool! This roughly makes one new add-on per week - despite this success, there are still plenty further add-ons in the review queue, so future releases are likely not to slow down much from this rate.
+
+Worth to mention are the integrations with two quite widely used systems (at least in Germany): [innogy](https://www.openhab.org/addons/bindings/innogysmarthome/) with a broad portfolio and [tado°](https://www.openhab.org/addons/bindings/tado/) with climate and heating devices. Unfortunately, both of them do not offer any local integration, so that offline situations can not be covered.
```

Comment:
@kaikreuzer Protip: when linking to internal pages use relative URLs like `[innogy](/addons/bindings/innogysmarthome/)`, so that the navigation occurs within the SPA (it feels much faster).
https://vuepress.vuejs.org/guide/markdown.html#internal-links